### ISSUE TITLE
[Question] How can I make console unsavable?  

### ISSUE THREAD

**Issue by arthurcaccavo** [dbeaver/dbeaver#34747]
I have the urge to press ctrl+s ever now and then because of muscle memory but when I open a console tab I explicitly don't want it to be saved. Especially automatically becoming a script when I opened it as a console.

It would be nice to have an option to disable saving on console. Even nicer if it also disable the pesky "*" indicator of unsaved changes, obviously it has unsaved changes they were never meant to be saved! Also I think it's one of the biggest triggers of me unconsciously clicking ctrl+s.

Another option could be to show a window asking If one really wants to transform a console to a script. The unpleasant need to close it every 5 seconds eventually would reverse pavlov my brain into not clicking ctrl+s in dbeaver I think.  

I just want a nice folder with meaningful scripts that I can rely on not a bunch of 'script-1, script-2, script-3, script-4, script-5..., script-21937' where I can never find what I need...

**Comment by E1izabeth** [comment]
Thank you for sharing your use case and create feature request.

**Comment by E1izabeth** [comment]
We discussed and decided that we want to make the following changes regarding this issue:
- dirty flag should not be set for Console
- add Confirmation before saving with ability to remember the choice and define behavior in preferences on Confirmations page

**Comment by arthurcaccavo** [comment]
this seems perfect! thank you!

**Comment by arthurcaccavo** [comment]
Hello!

Thanks for the work! I tested this today and it's just what I needed! I really love open source :) keep on the good work and community care, this is one project to make a change!

I would just like to confirm one detail:
> * dirty flag should not be set for Console

was this goal abandoned? Because I was able to set it to never save but the dirty flag is now always showing the moment I edit anything on a new console window. I am just confirming this as the status was set to close and I don't know if any further change is planned for this issue.
